anvil/src/winit.rs

```diff
@@ -94,7 +94,7 @@ pub fn run_winit() {
     let (mut backend, mut winit) = match winit::init::<GlesRenderer>() {
         Ok(ret) => ret,
         Err(err) => {
-            error!("Failed to initialize Winit backend: {}", err);
+            error!("Failed to initialize Winit backend: {:?}", err);
```

Comment:
This change seems unrelated? (Though a good one, that isn't worth it's own PR, please put it in a separate commit.)

Reply:
Whoops, that snuck in.